### ISSUE TITLE
[a11y] [Windows OS High Contrast Mode]: Checkbox not visible when unchecked

### ISSUE THREAD

**Issue by nkn2022** [carbon-design-system/carbon#16293]
### Package

carbon-components-react

### Browser

Chrome

### Operating System

Windows

### Package version

7.59.13

### React version

16.14.0

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker

### Description

Needs review
Confirm Windows high contrast mode is supported when using CSS to include, position or alter non-decorative content
Windows high contrast mode must be supported when using CSS to include, position or alter non-decorative content

Checkbox is not visible when unchecked by default. When checked it appears okay.
![image](https://github.com/carbon-design-system/carbon/assets/102628718/334c514a-01ca-495b-8255-0c6cd9265016)![image](https://github.com/carbon-design-system/carbon/assets/102628718/e61f9db8-b023-48bf-bec3-aff56e3ed21c)

### WCAG 2.1 Violation

1.1.1 Non-text Content

### Reproduction/example

NA

### Steps to reproduce

From the tool report:
> What to do
> On a Windows machine, activate high contrast mode
> confirm that checkboxes are visible with good contrast
> Rule ID: style_highcontrast_visible
> Reason ID: Manual_1

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by nkn2022** [comment]
This has been fixed after upgrade to Carbon 11. 
<img width="260" alt="image" src="https://github.com/user-attachments/assets/835fd9bd-22f2-4040-bc57-d99725354a90">

Closing this issue.
